ideascube/templates/ideascube/includes/cards/deus-ex-silicium.html

```diff
@@ -1,3 +1,2 @@
 {% load i18n %}
-{% trans "learn" as theme %}
-{% include 'ideascube/includes/cards/_kiwix.html' with card=card class="deus-ex-silicium" theme_class="learn" name="Deus Ex Silicium" description="Vidéos dédiées à la science des semi-conducteurs" theme=theme %}
+{% include 'ideascube/includes/cards/_kiwix.html' with card=card class="deus-ex-silicium" theme="learn" %}
```

Comment:
I agree that it would have been better to have it in a separate commit, and I'm sorry you spend more time on this than necessary

As you said, the two potential commits would change the same lines. It not as just split different chunks in different commits but rewrite all lines by hand.
If we consider that these files will be simply suppressed in a next future and that you already understand these changes, it is a lot of work for a very small improvement.